### PR TITLE
Adding Account details and projects for first time on ProjectHub

### PR DESCRIPTION
**Pull request by Panth1823**
## Screenshots

Screenshot

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
@Panth1823 add the issue number.

**Comment by Panth1823** [comment]
How to add an issue number? @priyankarpal

**Comment by priyankarpal** [comment]
> How to add an issue number? @priyankarpal

have you created an issue for this PR? @Panth1823

**Comment by Panth1823** [comment]
chore: project addition by [Panth1823] #1710

**Review by priyankarpal — CHANGES_REQUESTED** [review]
change the PR title to ` chore: project addition by Panth1823 #1710 `

File: DB/projects.json [hunk 1]
```diff
@@ -4026,4 +4026,26 @@
   ]
 }
 ]
```
Comment (priyankarpal) [inline DB/projects.json:4028]: remove
```suggestion

```

File: DB/projects.json [hunk 2]
```diff
@@ -4026,4 +4026,26 @@
   ]
 }
```
Comment (priyankarpal) [inline DB/projects.json:4027]: add a `,`
```suggestion
},
```

File: DB/projects.json [hunk 3]
```diff
@@ -4026,4 +4026,26 @@
...
+      "tech": ["reactjs", "tailwind"]
+    },
+    {
+      "link": "https://github.com/Panth1823/NexusNews.",
+      "title": "NexusNews",
+      "description": "NexusNews is a React-based news website that offers real-time updates, personalized news experience, and expert analysis with a clean and intuitive interface.",
+      "tech": ["reactjs", "css","json","nodejs"]
+    }
+  ]
+}
```
Comment (priyankarpal) [inline DB/projects.json:4051]: add `]`
```suggestion
}]
```

File: DB/projects.json [hunk 4]
```diff
@@ -4026,4 +4026,26 @@
...
+    "LinkedIn": "https://www.linkedin.com/in/panth1823/",
+    "Instagram": "https://www.instagram.com/_panth_18/?hl=en",
+    "Twitter": "https://twitter.com/Panth_1823"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/Panth1823/Textplorer-react",
+      "title": "Textplorer",
+      "description": "Textplorer is a React-based text manipulation web app that simplifies modifying text. It offers various features such as formatting, editing, and searching to boost productivity",
+      "tech": ["reactjs", "tailwind"]
```
Comment (priyankarpal) [inline DB/projects.json:4042]: rename to `tailwindcss`
```suggestion
      "tech": ["reactjs", "tailwindcss"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
> chore: project addition by [Panth1823] #1710

you have to add the issue number to the PR description. It's a best practice.

**Pull request closed without merging**